.github/workflows/master.yml

```diff
@@ -242,6 +242,14 @@ jobs:
       build_name: binary_s390x
       data: ${{ needs.RunConfig.outputs.data }}
       checkout_depth: 0
+  BuilderBinLOONGARCH64:
```

Comment:
Unrelated to this file:

To try the loongarch port, I ran locally

```
$ rm -rf build && cmake -DCMAKE_TOOLCHAIN_FILE=cmake/linux/toolchain-loongarch64.cmake -DENABLE_EMBEDDED_COMPILER=1 -DENABLE_LIBRARIES=1 -DCMAKE_BUILD_TYPE=Debug -DPARALLEL_COMPILE_JOBS=64 -DPARALLEL_LINK_JOBS=64 -S . -B build
```

which is the standard CMake invocation for cross-compiling. This was the output:

```
~/repo/ch6 (loongarch64 =) $ rm -rf build && cmake -DCMAKE_TOOLCHAIN_FILE=cmake/linux/toolchain-loongarch64.cmake -DENABLE_EMBEDDED_COMPILER=1 -DENABLE_LIBRARIES=1 -DCMAKE_BUILD_TYPE=Debug -DPARALLEL_COMPILE_JOB
S=64 -DPARALLEL_LINK_JOBS=64 -S . -B build
-- The C compiler identification is Clang 17.0.6
-- The CXX compiler identification is Clang 17.0.6
-- The ASM compiler identification is Clang with GNU-like command-line
-- Found assembler: /usr/bin/clang-17
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: /usr/bin/clang-17
-- Check for working C compiler: /usr/bin/clang-17 - broken
CMake Error at /usr/share/cmake-3.22/Modules/CMakeTestCCompiler.cmake:69 (message):
  The C compiler

    "/usr/bin/clang-17"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /home/ubuntu/repo/ch6/build/CMakeFiles/CMakeTmp

    Run Build Command(s):/usr/bin/ninja cmTC_d8b4c && [1/2] Building C object CMakeFiles/cmTC_d8b4c.dir/testCCompiler.c.o
    FAILED: CMakeFiles/cmTC_d8b4c.dir/testCCompiler.c.o
    /usr/bin/clang-17 --target=loongarch64-linux-gnu --sysroot=/home/ubuntu/repo/ch6/cmake/linux/../../contrib/sysroot/linux-loongarch64   -mcmodel=extreme --gcc-toolchain=/home/ubuntu/repo/ch6/cmake/linux/../..
/contrib/sysroot/linux-loongarch64/usr -MD -MT CMakeFiles/cmTC_d8b4c.dir/testCCompiler.c.o -MF CMakeFiles/cmTC_d8b4c.dir/testCCompiler.c.o.d -o CMakeFiles/cmTC_d8b4c.dir/testCCompiler.c.o -c /home/ubuntu/repo/ch
6/build/CMakeFiles/CMakeTmp/testCCompiler.c
    clang-17: error: invalid argument 'extreme' to -mcmodel=
    ninja: build stopped: subcommand failed.





  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:3 (project)
```